README.rst

```diff
@@ -0,0 +1,54 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2015 CERN.
+
+    Invenio is free software; you can redistribute it
+    and/or modify it under the terms of the GNU General Public License as
+    published by the Free Software Foundation; either version 2 of the
+    License, or (at your option) any later version.
+
+    Invenio is distributed in the hope that it will be
+    useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+    General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Invenio; if not, write to the
+    Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+    MA 02111-1307, USA.
+
+    In applying this license, CERN does not
+    waive the privileges and immunities granted to it by virtue of its status
+    as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+======================================
+Invenio Schema
+======================================
+
+.. image:: https://img.shields.io/travis/inveniosoftware/invenio-schema.svg
+        :target: https://travis-ci.org/inveniosoftware/invenio-schema
+
+.. image:: https://img.shields.io/coveralls/inveniosoftware/invenio-schema.svg
+        :target: https://coveralls.io/r/inveniosoftware/invenio-schema
+
+.. image:: https://img.shields.io/github/tag/inveniosoftware/invenio-schema.svg
+        :target: https://github.com/inveniosoftware/invenio-schema/releases
+
+.. image:: https://img.shields.io/pypi/dm/invenio-schema.svg
+        :target: https://pypi.python.org/pypi/invenio-schema
+
+.. image:: https://img.shields.io/github/license/inveniosoftware/invenio-schema.svg
+        :target: https://github.com/inveniosoftware/invenio-schema/blob/master/LICENSE
+
+
+Invenio Schema Registry
```

Comment:
Considering that the JSON Schema IETF draft itself uses "schemas", OK to go for `invenio-json-schemas`, with or without hyphen.  

P.S. Do you also prefer "radiuses" instead of "radii"? :)